src/ui/public/fixed_scroll.js

```diff
@@ -98,16 +103,28 @@ uiModules
         listen();
       }
 
-      // reset when the width or scrollWidth of the $el changes
-      $scope.$watchMulti([
-        function () { return $el.prop('scrollWidth'); },
-        function () { return $el.width(); }
-      ], setup);
+      let width;
+      let scrollWidth;
+      let widthCheckTimeout = $timeout(checkWidth, 500);
```

Comment:
I thought about window resize, but there are other situations where the container width might change, like adding/removing columns from the doc table. I also tried the scroll event, thinking it might fire when the container changes size since the scroll bar would also be changing size, but no dice :(  

afaik there is no event that'll tell us when the container has changed size 100% of the time